src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -109,6 +111,11 @@ public OtlpExporterOptions()
         /// </summary>
         public OtlpExportProtocol Protocol { get; set; } = OtlpExportProtocol.Grpc;
 
+        /// <summary>
+        /// Gets metric-specific OTLP exporter options.
+        /// </summary>
+        public OtlpMetricExporterOptions MetricOptions { get; }
```

Comment:
Hmm, just occurred to me... I don't know if this plays well with the DeferredMeterProvider and pulling options from the service provider.